config/200-clusterrole.yaml

```diff
@@ -10,3 +10,18 @@ rules:
   - apiGroups: ["projectcontour.io"]
     resources: ["httpproxies"]
     verbs: ["get", "list", "create", "update", "delete", "deletecollection", "patch", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
We actually shouldn't need this 🤔 

In Serving we use an [aggregated cluster role](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) [here](https://github.com/knative/serving/blob/89d45259bbc65a29f142d3c7d6179a86cc865477/config/core/200-serviceaccount.yaml#L23-L33) which should pick up the role above simply by having it labelled.

Reply:
didn't know aggregated cluster role. That's very neat. This PR can be closed.